[gsettings_desktop_schemas] Failed to build

```
[2/4] Generating GDesktopEnums-3.0.gir with a custom command
FAILED: headers/GDesktopEnums-3.0.gir 
/usr/local/bin/g-ir-scanner --no-libtool --namespace=GDesktopEnums --nsversion=3.0 --warn-all --output headers/GDesktopEnums-3.0.gir --header-only -I/usr/local/tmp/crew/3.38.0.tar.gz.dir/gsettings-desktop-schemas-3.38.0/headers -I/usr/local/tmp/crew/3.38.0.tar.gz.dir/gsettings-desktop-schemas-3.38.0/builddir/headers --filelist=/usr/local/tmp/crew/3.38.0.tar.gz.dir/gsettings-desktop-schemas-3.38.0/builddir/headers/libnoinst.so.p/GDesktopEnums_3.0_gir_filelist --identifier-prefix=GDesktop --cflags-begin -I/usr/local/include/gobject-introspection-1.0 -I/usr/local/lib64/libffi-3.2.1/include -I/usr/local/include/glib-2.0 -I/usr/local/lib64/glib-2.0/include --cflags-end --add-include-path=/usr/local/share/gir-1.0 --library noinst -L/usr/local/tmp/crew/3.38.0.tar.gz.dir/gsettings-desktop-schemas-3.38.0/builddir/headers -lgirepository-1.0 -lgobject-2.0 -lglib-2.0
Traceback (most recent call last):
  File "/usr/local/bin/g-ir-scanner", line 98, in <module>
    from giscanner.scannermain import scanner_main
  File "/usr/local/lib64/gobject-introspection/giscanner/scannermain.py", line 35, in <module>
    from giscanner.ast import Include, Namespace
  File "/usr/local/lib64/gobject-introspection/giscanner/ast.py", line 29, in <module>
    from .sourcescanner import CTYPE_TYPEDEF, CSYMBOL_TYPE_TYPEDEF
  File "/usr/local/lib64/gobject-introspection/giscanner/sourcescanner.py", line 35, in <module>
    from giscanner._giscanner import SourceScanner as CSourceScanner
ModuleNotFoundError: No module named 'giscanner._giscanner'
[3/4] Generating org.gnome.desktop.enums.xml with a custom command (wrapped by meson to capture output)
ninja: build stopped: subcommand failed.
gsettings_desktop_schemas failed to build: `ninja -C builddir` exited with 1
```

--- a/packages/gsettings_desktop_schemas.rb
+++ b/packages/gsettings_desktop_schemas.rb
@@ -8,6 +8,19 @@ class Gsettings_desktop_schemas < Package
   source_url 'https://github.com/GNOME/gsettings-desktop-schemas/archive/3.38.0.tar.gz'
   source_sha256 'b808bd285ac7176f2e9e3a8763c3913121ab9f109d2988c70e3f1f8e742a630d'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gsettings_desktop_schemas-3.38.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gsettings_desktop_schemas-3.38.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gsettings_desktop_schemas-3.38.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gsettings_desktop_schemas-3.38.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'dd97405ae148e26b65dfce4f91711dd73c54a021fc8d1f3463c9af30c1874d82',
+     armv7l: 'dd97405ae148e26b65dfce4f91711dd73c54a021fc8d1f3463c9af30c1874d82',
+       i686: '87eb145275e5d3c3bbb2042b6dd0bfd334012d3665b086dd1c4c722d10510287',
+     x86_64: '69e40284e42d687b48fd5c35ef84d695a94ba0e8483ce2cb0654cc07886a177b',
+  })
+
   depends_on 'gnome_common'
   depends_on 'glib'
   depends_on 'gobject_introspection'

--- a/packages/gspell.rb
+++ b/packages/gspell.rb
@@ -7,6 +7,19 @@ class Gspell < Package
   source_url 'https://download.gnome.org/sources/gspell/1.9/gspell-1.9.1.tar.xz'
   source_sha256 'dcbb769dfdde8e3c0a8ed3102ce7e661abbf7ddf85df08b29915e92cd723abdd'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gspell-1.9.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gspell-1.9.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gspell-1.9.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gspell-1.9.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'b804ca54e3ff7f007d078c1a3f329b67131f7e4fe2fb72d835c2d93762093c7b',
+     armv7l: 'b804ca54e3ff7f007d078c1a3f329b67131f7e4fe2fb72d835c2d93762093c7b',
+       i686: 'b05c30504de2f207fca5f4ff92fff5294db9d1479b0404028b5345c972cea38e',
+     x86_64: 'f7c41e8c5b900aa694b002f5134b89d9c9757e7e5f56c060b7f308fb146dd5dc',
+  })
+
   depends_on 'gtk3'
   depends_on 'enchant'
   depends_on 'libxml2'

--- a/packages/libsecret.rb
+++ b/packages/libsecret.rb
@@ -8,6 +8,18 @@ class Libsecret < Package
   source_url 'https://github.com/GNOME/libsecret/archive/0.20.4.tar.gz'
   source_sha256 'ca34e69b210df221ae5da6692c2cb15ef169bb4daf42e204442f24fdb0520d4b'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsecret-0.20.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsecret-0.20.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsecret-0.20.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsecret-0.20.4-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '72e98cbae2d269061a3d2ca137a8437642d9cc72a130f02fecddd405dd968c95',
+     armv7l: '72e98cbae2d269061a3d2ca137a8437642d9cc72a130f02fecddd405dd968c95',
+       i686: '3a58950489de1b2dc1cdb57c81f73b6aca116dac47e13ee486c0bbf513c54dff',
+     x86_64: '8d4f30362b10c787837da9ed3568d27933f22cea9f6b605ad5f8fec20b5e755d',
+  })
 
   depends_on 'gobject_introspection' => :build
   depends_on 'libgcrypt' => :build

--- a/packages/pkgconf.rb
+++ b/packages/pkgconf.rb
@@ -8,6 +8,19 @@ class Pkgconf < Package
   source_url 'https://github.com/pkgconf/pkgconf/archive/pkgconf-1.7.3.tar.gz'
   source_sha256 '8f2c6e9f08adc5773d7fa3c1db1ed03f5fa02ceed037a537ce1195f7c93700ed'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/pkgconf-1.7.3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/pkgconf-1.7.3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/pkgconf-1.7.3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/pkgconf-1.7.3-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '7dba3ff97aa46e3eb5ca3a3874b10d0d7952cbbd655c94d090da59827f3c05c3',
+     armv7l: '7dba3ff97aa46e3eb5ca3a3874b10d0d7952cbbd655c94d090da59827f3c05c3',
+       i686: 'fb51ba2693426e45f973cac8f053556c27864e0e1c4b3a08f29f054a77dac800',
+     x86_64: '014ca1e27dae6c162677a12ed73138631bb81f3749cfe093987208f84eaebcf1',
+  })
+
   depends_on 'glibc'
 
 # Can be enabled for packages by setting

--- a/packages/xterm.rb
+++ b/packages/xterm.rb
@@ -8,7 +8,18 @@ class Xterm < Package
   source_url "https://invisible-mirror.net/archives/xterm/xterm-362.tgz"
   source_sha256 '1d4ffe226fa8f021859bbc3007788ff63a46a31242d9bd9a7bd7ebe24e81aca2'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/xterm-362-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/xterm-362-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/xterm-362-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/xterm-362-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'e61f8a1bfc06e700ec42b0d3f7e97b9b1e0f8120a33fbfc6efeba3eabda24126',
+     armv7l: 'e61f8a1bfc06e700ec42b0d3f7e97b9b1e0f8120a33fbfc6efeba3eabda24126',
+       i686: '7912500763cca0d85c1f6d84ec4cce6cbd5b4c9a70090dfe49cc94a395b7b84b',
+     x86_64: 'c6dd0a0eebbe9a84af571c79ad1ecb925b1a16333bb6e75280f5818c86faedb5',
+  })
 
   depends_on 'pcre'
   depends_on 'sommelier'

